app/src/components/items/uv/universalViewer.tsx

```diff
@@ -14,9 +14,25 @@ export type UniversalViewerProps = {
   onChangeManifest?: (manifest: string) => void;
 };
 
+const handleOnClick = (e) => {
+  if (e.target.className === "openseadragon-canvas") {
+    console.log("target is image viewer");
+    // this doesn't work. think we need to select the great-grandparent? div
```

Comment:
curious what exactly still isn't working here - is it the upstream seadragon bug?  If so, might be helpful to comment something more explicit.

Reply:
ope. need to remove this comment!